tests/driver_sx127x/main.c

```diff
@@ -376,7 +376,7 @@ void *_recv_thread(void *arg)
 
 int main(void)
 {
-    memcpy(&sx127x.params, sx127x_params, sizeof(sx127x_params));
+    sx127x.params = *sx127x_params;
```

Comment:
maybe use `sx127x.params = sx127x_params[0];` as the latter is an array defined in the included header file. This assignment is a bit different from the other driver init functions where only one element is passed in.

Reply:
at least that would be (more) consistent with the changes to usage of `phydat_t` (`res->val` vs. `res->val[0]`) in SAUL, see some of your other (merged) PRs.

Reply:
Good point. Thanks for noting, @smlng, with an 'l' :-)
So, in this case it was actually a bug that got fixed.